### ISSUE TITLE
Wrong outputs in GroupNorm function

### ISSUE THREAD

**Issue by f90** [pytorch/pytorch#22617]
## 🐛 Bug

Output of ``nn.GroupNorm`` is not correct (not the same as the result obtained when using the equation from the docs/paper)

## To Reproduce

```
import torch
x = torch.rand((1,1,5)) # Batch size of 1, 1 channel, 5 entries to normalise
gn = torch.nn.GroupNorm(1,1, eps=1e-5)

manual_norm = (x - torch.mean(x)) / torch.sqrt(torch.var(x) + 1e-5)
manual_norm = manual_norm * gn.weight + gn.bias

print(gn(x))
print(manual_norm)

print(torch.var(gn(x)))
print(torch.var(manual_norm))
```

## Expected behavior

GroupNorm function output ``gn(x)`` should be equal to manually normalised output ``manual_norm``, but instead it is different. Also the variances (the last two numbers)  indicate that GroupNorm does not normalise properly to unit variance, but my manual implementation does:

```
tensor([[[-1.3768, -0.3985, -0.5176,  0.9900,  1.3028]]],
       grad_fn=<AddcmulBackward>)
tensor([[[-1.2314, -0.3564, -0.4629,  0.8855,  1.1653]]],
       grad_fn=<AddBackward0>)
tensor(1.2499, grad_fn=<VarBackward0>)
tensor(0.9999, grad_fn=<VarBackward0>)
```

## Environment

```
Collecting environment information...
PyTorch version: 1.1.0
Is debug build: No
CUDA used to build PyTorch: 9.0.176

OS: Linux Mint 19.1 Tessa
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: Could not collect

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: GeForce GTX 1080
Nvidia driver version: 390.116
cuDNN version: Could not collect

Versions of relevant libraries:
[pip3] numpy==1.16.2
[pip3] torch==1.1.0
[pip3] torchsummary==1.5.1
[pip3] torchtext==0.3.1
[pip3] torchvision==0.3.0
[conda] Could not collect
```

**Comment by f90** [comment]
My bad, it seems that it uses the unbiased variance, so if you add ``unbiased=False`` to all ``torch.var`` commands in the example code, it works out... Might be good to note in the docs though to avoid confusion!
